src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -44,6 +44,10 @@ export function statementOfTruthFullName(formData, fullNamePath) {
   return fullNameString;
 }
 
+function fullNameReducer(fullNameString) {
+  return fullNameString.replaceAll(' ', '').toLowerCase();
```

Comment:
is there any chance of fullNameString being null here? like maybe if they refreshed on the review page or something? if so we could get a NPE 👀 maybe adding a fullNameString?.replaceAll(...)?.toLowerCase() ?  or am I too paranoid about this type of thing? :)

Reply:
That's actually a really good question. I didn't have any issues in testing, but it does seem like that might happen. I guess it would only be at the beginning when you first type? Because after that, it would always go back to `''` empty string if you deleted everything. It does seem like it would be undefined/null before anything is typed though.

Maybe because it's mapped to a value in the schema that value is somehow defaulting to empty string? Not sure, but you've got me thinking haha. I'll probably just change it to your suggestion anyway because there's no harm in it.